S/Skia/build_tarballs.jl

```diff
@@ -0,0 +1,101 @@
+dependencies = [
+    Dependency("Fontconfig_jll"; compat="2.16.0")
+    Dependency(PackageSpec(name="Xorg_libX11_jll", uuid="4f6342f7-b3d2-589e-9d20-edeb45f2b2bc");)
+    Dependency(PackageSpec(name="xkbcommon_jll", uuid="d8fb68d0-12a3-5cfd-a85a-d49703b185fd"); )
+    Dependency(PackageSpec(name="Libglvnd_jll", uuid="7e76a0d4-f3c7-5321-8279-8d96eeed0f29"); )
+    BuildDependency(PackageSpec(name="Xorg_xorgproto_jll", uuid="c4d99508-4286-5418-9131-c86396af500b");)
+
+]
+
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+
+cd $WORKSPACE/srcdir/
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd skia
+
+install_license LICENSE 
+
+bin/fetch-gn 
+python3 tools/git-sync-deps || true
+
+cp ../cskia/capi/sk_capi.cpp src/base/
+cp ../cskia/capi/sk_capi.h src/base/
+
+if [[ "${target}" == x86_64-* ]]; then
+bin/gn gen out/Static --args='target_cpu="x64" cc="clang" cxx="clang++" is_official_build=true skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_vulkan=true skia_use_system_freetype2=false skia_use_fontconfig=true skia_enable_pdf=true  skia_use_system_icu=false skia_use_system_expat=false skia_use_harfbuzz=false skia_use_vulkan=true skia_use_gl=true extra_cflags=["-fpic", "-fvisibility=default"]'
+fi
+
+if [[ "${target}" == aarch64-* ]]; then
+bin/gn gen out/Static --args='target_cpu="arm64" cc="clang" cxx="clang++" is_official_build=true skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_vulkan=true skia_use_system_freetype2=false skia_use_fontconfig=true skia_enable_pdf=true  skia_use_system_icu=false skia_use_system_expat=false skia_use_harfbuzz=false skia_use_vulkan=true skia_use_gl=true extra_cflags=["-fpic", "-fvisibility=default"]'
+
+fi
+
+if [[ "${target}" == riscv64-* ]]; then
+bin/gn gen out/Static --args='target_cpu="riscv" cc="clang" cxx="clang++" is_official_build=true skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_vulkan=true skia_use_system_freetype2=false skia_use_fontconfig=true skia_enable_pdf=true  skia_use_system_icu=false skia_use_system_expat=false skia_use_harfbuzz=false skia_use_vulkan=true skia_use_gl=true extra_cflags=["-fpic", "-fvisibility=default"]'
+fi
+
+if [[ "${target}" == powerpc64le-* ]]; then
+bin/gn gen out/Static --args='target_cpu="powerpc64le" cc="clang" cxx="clang++" is_official_build=true skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_vulkan=true skia_use_system_freetype2=false skia_use_fontconfig=true skia_enable_pdf=true  skia_use_system_icu=false skia_use_system_expat=false skia_use_harfbuzz=false skia_use_vulkan=true skia_use_gl=true extra_cflags=["-fpic", "-fvisibility=default"]'
+fi
```

Comment:
Is the only difference between these commands the `target_cpu` argument that is being passed? If so, I think this is much better done by doing an if/elseif to set one variable with the CPU name, and then used that variable in a command after it to actually do the call to the build system. (and even if there is another command or two that change value, I think it is still better to do it this way).

Also, please break each build option into its own line. It is very hard to keep track of the options now, and in the future if changes have to be made, having them on their own lines allows better tracking of changes in the git history.